website/client/src/components/challenges/challengeDetail.vue

```diff
@@ -401,6 +461,24 @@ export default {
     canJoin () {
       return !this.isMember;
     },
+    // canViewFlags should allow only moderators/admins to see flags
+    canViewFlags () {
+      const isModerator = this.hasPermission(this.user, 'moderator');
+      if (isModerator && this.challenge.flagCount > 0) return true;
+      return false;
+    },
+    // flaggedNotHidden should allow mods/admins & challenge owner to see flags
+    flaggedNotHidden () {
+      return this.challenge.flagCount === 1;
+    },
+    // flaggedAndHidden should only allow admin to see challenge & flags
+    flaggedAndHidden () {
+      return this.challenge.flagCount > 1;
+    },
+    // isFlagged disables the clone button
+    isFlagged () {
+      return this.challenge.flagCount > 1;
```

Comment:
This is identical to the logic for `flaggedAndHidden`, is it supposed to be > 0? or should we just combine the two?